benchmark_v2/io_benchmark.py

```diff
@@ -0,0 +1,259 @@
+import os
+from glob import glob
+
+import pytest
+
+import arkouda as ak
+
+TYPES = ("int64", "float64", "uint64", "str")
+FILETYPES = ("HDF5", "PARQUET")
+COMPRESSIONS = ("none", "snappy", "gzip", "brotli", "zstd", "lz4")

```

Comment:
I believe `"none"` should be `None` here based on how `aroukda.io.to_parquet` is configured.

Reply:
Updated